blog_system/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.urls import path, include
+from debug_toolbar.toolbar import debug_toolbar_urls
+
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("blog/", include("blog.urls", namespace="blog")),
+    path("", include("django.contrib.auth.urls")),
+] + debug_toolbar_urls()
```

Comment:
The 'debug_toolbar_urls()' should be included conditionally, typically only in development mode. Consider wrapping this with a condition to check if the application is in DEBUG mode.